feature/bitwarden/feature.yaml

```diff
@@ -0,0 +1,2 @@
+title: "🔐 Bitwarden"
+description: "Installs the [Bitwarden cli](https://bitwarden.com/help/cli/)"
```

Comment:
just cosmetics: i prefer cli in capital letters

Reply:
Fixed